CHANGELOG.rst

```diff
@@ -18,7 +18,7 @@ Added
 - Added listener for ``kytos/core.interface_tags`` event to save any changes made to ``Interface`` attributes ``tag_ranges`` and ``available_tags``
 - Added script ``special_vlan_allocation.py`` to add ``special_available_tags`` and ``special_tags`` fields to ``interface_details`` collection.
 - Added endpoint ``POST v3/interfaces/{interface_id}/special_tags`` to set ``special_tags`` to interfaces.
-- Added endpoint ``DELETE v3/links/{link_id}`` to delete a disabled link.
+- Added endpoint ``DELETE v3/links/{link_id}`` to delete a disabled link. To avoid rediscovery, the link needs to be physically disconnected or both interface ends (switches) need to be disabled.
```

Comment:
``DELETE /v3/switches/{dpid}`` hasn't been included in the changelog. 

It'll be great to also add a similar heads up regarding discovery. Once the switch is deleted, network operators should remove in the physical/virtual switch the configuration of setting a controller to Kytos, otherwise after a reconnection, it'll get rediscovered again. Let's explain this and make it clear that the purpose of the deletion here is mostly for decommissioning